src/angularD3/directives/data.ts

```diff
@@ -38,7 +38,7 @@ export class D3Data implements AfterContentInit {
   fetch() {
     if (!this._src) throw('No src url given')
     this._d3Data.csv(this._src, this.accessor || this.defaultFilter, this.callback)
-      .then((rows) => { this.data.next({ rows: rows }) })
+      .then((rows) => { this.data.emit({ rows: rows }) })
```

Comment:
Trying out this change to see if it makes any different. It didn't. Not sure if `next` or `emit` is the right approach here.